keycloak-js/index.d.ts

```diff
@@ -0,0 +1,447 @@
+// Type definitions for keycloak-js 2.4
+// Project: https://github.com/keycloak/keycloak
```

Comment:
Here's the documentation: https://keycloak.gitbooks.io/securing-client-applications-guide/content/topics/oidc/javascript-adapter.html

I added links to the JSDoc comments on the constructor and the instance interface.